src/java.base/share/classes/jdk/internal/misc/Unsafe.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2000, 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2000, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Thanks for this update; thought we don't update headers in valhalla.

Reply:
For OpenJDK projects, copyrights are updated. Do it now, you won't have to do it later.